tests/financial/test_bolsas_y_mercados_espanoles.py

```
@@ -0,0 +1,204 @@
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.financial.bolsas_y_mercados_espanoles import BolsasYMercadosEspanoles
+from tests.common import CommonFinancialTests
+
+
+class TestBolsasYMercadosEspanoles(CommonFinancialTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.21)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)